### PR TITLE
Minor tidying of the Consul documentation

### PR DESCRIPTION
**Pull request by jamtur01**

### CODE REVIEW

File: website/source/docs/providers/consul/d/keys.html.markdown [hunk 1]
```diff
@@ -8,8 +8,9 @@ description: |-
 
 # consul\_keys
 
-`consul_keys` reads values from the Consul key/value store.
-This is a powerful way dynamically set values in templates.
+The `consul_keys` resource reads values from the Consul key/value store.
+This is a powerful way to  This is a powerful way dynamically set values
```
Comment (Ninir) [inline website/source/docs/providers/consul/d/keys.html.markdown]: Small repetition there ;-)

File: website/source/docs/providers/consul/d/keys.html.markdown [hunk 2]
```diff
@@ -8,8 +8,9 @@ description: |-
...
+in templates.dynamically set values in templates.
```
Comment (Ninir) [inline website/source/docs/providers/consul/d/keys.html.markdown]: Again there! 😄

File: website/source/docs/providers/consul/r/catalog_entry.html.markdown [hunk 3]
```diff
@@ -30,14 +31,14 @@ resource "consul_catalog_entry" "app" {
 
 The following arguments are supported:
 
-* `address` - (Required) The address of the node being added to
+* `address` - (Required) The address of the node being added to,
   or referenced in the catalog.
 
-* `node` - (Required) The name of the node being added to or
+* `node` - (Required) The name of the node being added to or,
```
Comment (Ninir) [inline website/source/docs/providers/consul/r/catalog_entry.html.markdown]: Shouldn't this be after the `to` as exposed above?

**Comment by jamtur01** [comment]
Thanks @Ninir Addressed.

**Comment by mitchellh** [comment]
Great! Thanks.

**Pull request merged**